packages/turbo-repository/scripts/bump-version.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+set -e
+
+if [ $# != 1 ]; then
+  echo "Missing version"
+  exit 1
+fi
+TARGET=$1
+
+SCRIPTS_DIR="$( dirname -- "${BASH_SOURCE[0]}" )"
+PKG_ROOT=$(realpath "${SCRIPTS_DIR}/..")
```

Comment:
how come you used dirname in one line and `/..` in another? could just do `/../..`

Reply:
Heh, the first line was copied from a "get directory of script" SO answer.

I do think there's some slight value in keeping it two lines, which shows the steps for the calculation, but I'll update the second line to use `dirname` instead of `realpath` for some consistency.

Reply:
Ok, ended up moving both of them to `realpath` instead, since it turns out `BASH_SOURCE[0]` is a _relative_ path. I left a note to that effect as well.